### DISCUSSION TITLE
cm3-unix64le-d5.11.1-20210610 built successfully for AARCH64 Linux

### DISCUSSION THREAD

**Discussion by Pawlicker** [modula3/cm3#770] [Show and tell]
I've been a bit busy with real life stuff but I've managed to compile cm3-unix64le-d5.11.1-20210610 on an RPI4 running Ubuntu 20.04 and made the distro. For some reason it failed when making the deb file but it was able to successfully make the tgz file.

I'm not sure how else to share it, so here's the file that I've uploaded to my VPS. 
https://www.pawlicker.com/files/modula3/cm3-all-ARM64_LINUX-d5.11.1-Linux5.4.0-20211012.tar.gz

**Comment by ghost** [comment]
Thanks!  I've added it to our releases page https://github.com/modula3/cm3/releases/tag/d5.11.3

It's perhaps a little miscategorized, but at least it's up there if someone goes looking for it.

**Comment by MarcusE1W** [comment]
Happy to report that I could build D5.11.4 on the Pinebook Pro with aarch64 and Manjaro-ARM.

the parameter ` --target ARM64_LINUX` was required tough #805

**Reply by ghost** [reply]
Thanks for trying that out.  I've updated the getting started instructions on the wiki.
